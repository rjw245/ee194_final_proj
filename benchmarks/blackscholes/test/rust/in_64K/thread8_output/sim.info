{'cmdline': ['/h/hzhou01/comp140/sniper/run-sniper',
             '-d',
             './thread8_output',
             '-n',
             '9',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '8',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell24',
 'pin_version': 71313,
 'rusage': (56253.556159,
            731.451802,
            205808,
            0,
            0,
            0,
            446429,
            0,
            0,
            0,
            12656,
            0,
            0,
            0,
            53170821,
            182143),
 'snipercmd': ['bash',
               '-c',
               '/h/hzhou01/comp140/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/hzhou01/comp140/sniper/lib/pin_sim -c /h/hzhou01/comp140/sniper/config/base.cfg --general/total_cores=9 --general/output_dir=/h/hzhou01/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread8_output --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg --config=/h/hzhou01/comp140/sniper/config/gainestown.cfg --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 8 ../../../inputs/in_64K.txt'],
 't_elapsed': 21449.771764039993,
 't_start': 1462232588.125545,
 'tracecmds': [],
 'user': 'hzhou01',
 'vmem': 4159574016L}
