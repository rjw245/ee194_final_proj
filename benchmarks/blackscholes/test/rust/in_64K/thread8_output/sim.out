                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7      | Core 8     
  Instructions                       |  4219276560 |  1663632199 |  1663648857 |  1663609724 |  1663680321 |  1663656617 |  1663646802 |  1663633423 |  1663639497
  Cycles                             |  3876925262 |  3876925262 |  3876925262 |  3876925262 |  3876925262 |  3876925262 |  3876925262 |  3876925262 |  3876925262
  IPC                                |        1.09 |        0.43 |        0.43 |        0.43 |        0.43 |        0.43 |        0.43 |        0.43 |        0.43
  Time (ns)                          |  1457490700 |  1457490700 |  1457490700 |  1457490700 |  1457490700 |  1457490700 |  1457490700 |  1457490700 |  1457490700
  Idle time (ns)                     |   525385953 |   932086253 |   936231929 |   932572462 |   937127519 |   936998605 |   936604221 |   936834800 |   936592469
  Idle time (%)                      |       36.0% |       64.0% |       64.2% |       64.0% |       64.3% |       64.3% |       64.3% |       64.3% |       64.3%
Branch predictor stats               |             |             |             |             |             |             |             |             |            
  num correct                        |   148350208 |    58823498 |    58841484 |    58635297 |    58861703 |    58859370 |    58846808 |    58848561 |    58846244
  num incorrect                      |     5796885 |     1536735 |     1518095 |     1723094 |     1499168 |     1499496 |     1513317 |     1509898 |     1513408
  misprediction rate                 |       3.76% |       2.55% |       2.52% |       2.85% |       2.48% |       2.48% |       2.51% |       2.50% |       2.51%
  mpki                               |        1.37 |        0.92 |        0.91 |        1.04 |        0.90 |        0.90 |        0.91 |        0.91 |        0.91
TLB Summary                          |             |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |             |            
    num accesses                     |   903867648 |   273098872 |   273100363 |   273096055 |   273102437 |   273099028 |   273101896 |   273096908 |   273100802
    num misses                       |         193 |          76 |          71 |          71 |          76 |          71 |          71 |          71 |          71
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |             |            
    num accesses                     |  2813843749 |   956470625 |   956486251 |   956458450 |   956504715 |   956491583 |   956479228 |   956478621 |   956476560
    num misses                       |      243511 |        2101 |        2328 |        2330 |        2368 |        2036 |        2335 |        2038 |        2349
    miss rate                        |       0.01% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.06 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |             |            
    num accesses                     |      243704 |        2177 |        2399 |        2401 |        2444 |        2107 |        2406 |        2109 |        2420
    num misses                       |         881 |         243 |         193 |         200 |         228 |         196 |         203 |         200 |         201
    miss rate                        |       0.36% |      11.16% |       8.05% |       8.33% |       9.33% |       9.30% |       8.44% |       9.48% |       8.31%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |             |            
    num cache accesses               |   903867648 |   273098872 |   273100363 |   273096055 |   273102437 |   273099028 |   273101896 |   273096908 |   273100802
    num cache misses                 |    22987767 |        1831 |        1334 |        2127 |        1369 |        1320 |        2134 |        1317 |        1726
    miss rate                        |       2.54% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        5.45 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Cache L1-D                         |             |             |             |             |             |             |             |             |            
    num cache accesses               |  2813844509 |   956470884 |   956486528 |   956458745 |   956505026 |   956491898 |   956479553 |   956478952 |   956476865
    num cache misses                 |      103143 |      344155 |      341916 |      343422 |      341911 |      343493 |      343523 |      342231 |      342222
    miss rate                        |       0.00% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04%
    mpki                             |        0.02 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21
  Cache L2                           |             |             |             |             |             |             |             |             |            
    num cache accesses               |    23091090 |      345993 |      343265 |      345553 |      343325 |      344829 |      345673 |      343558 |      343987
    num cache misses                 |       39564 |       12470 |        8197 |       15159 |        8341 |        8183 |       15154 |        8188 |       11687
    miss rate                        |       0.17% |       3.60% |       2.39% |       4.39% |       2.43% |       2.37% |       4.38% |       2.38% |       3.40%
    mpki                             |        0.01 |        0.01 |        0.00 |        0.01 |        0.01 |        0.00 |        0.01 |        0.00 |        0.01
  Cache L3                           |             |             |             |             |             |             |             |             |            
    num cache accesses               |       39760 |       12478 |        8212 |       15163 |        8386 |        8199 |       15170 |        8198 |       11726
    num cache misses                 |       33969 |         690 |         588 |         430 |        4648 |        3593 |        3537 |        3523 |        4739
    miss rate                        |      85.44% |       5.53% |       7.16% |       2.84% |      55.43% |      43.82% |      23.32% |      42.97% |      40.41%
    mpki                             |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
DRAM summary                         |             |             |             |             |             |             |             |             |            
  num dram accesses                  |       12824 |           0 |           0 |           0 |       12841 |           0 |           0 |           0 |       12893
  average dram access latency (ns)   |       55.92 |         inf |         inf |         inf |       55.69 |         inf |         inf |         inf |       55.95
  average dram queueing delay        |        2.49 |        0.00 |        0.00 |        0.00 |        2.26 |        0.00 |        0.00 |        0.00 |        2.52
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |             |            
  num loads from dram                |        3847 |         171 |          63 |          64 |          62 |          31 |          41 |          33 |         362
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |          33 |          11 |          48 |          26 |        3389 |        3132 |        3149 |        3143 |        3158
